schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id GENERATED ALWAYS AS IDENTITY, 
+    name varchar(100),
+    date_of_birth date,
+    escape_attempts INT,
+    neutered boolean,
+    weight_kg decimal
 );
```

Comment:
Optional  
   you can make the id column `NOT NULL` and autoincremented `IDENTITY(1,1)` so your table can be numbered and organized 👍🏾